config.json

```diff
@@ -16,6 +16,16 @@
       ],
       "auto_approve": true
     },
+    {
+      "slug": "reverse-string",
+      "uuid": "0000e75e-9b43-11e8-98d0-529269fb1459",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
Is this deliberately a 'bonus exercise' or is it worth specifying an `unlocked_by`?

Reply:
I think it's better to make it a bonus exercise no ?